src/components/dev-hub/contents-menu.js

```diff
@@ -54,27 +67,32 @@ const Contents = styled('ul')`
  * @property {object[]} props.contents
  */
 
-const ContentsMenu = ({ title, contents = [] }) => {
+const ContentsMenu = ({ title, headingNodes, ...props }) => {
```

Comment:
Did we get these ContentsMenu styles from anywhere?

Reply:
They are based on the designs if thats what you mean?

Reply:
Oo I see it now! Thanks!

Reply:
going to update the styles